sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -35,6 +35,13 @@ final class DataFrameStatFunctions private[sql](df: DataFrame) {
    * @param col2 the name of the second column
    * @return the covariance of the two columns.
    *
+   * {{{
+   *    import org.apache.spark.sql.functions._
+   *    val df = sc.parallelize(0 until 10).toDF("id").withColumn("rand1", rand(seed=10))
```

Comment:
had to change this to `sc.parallelize(...).toDF(...)` because `sqlContext.createDataFrame(0 until 10)` doesn't work because it needs to be a tuple I guess.